react/training-project/src/app/screens/Game/index.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+/* eslint-disable import/order */
```

Comment:
Why are you needing this? :thinking:

Reply:
The linter is bothering me when I use simultaneous alias import without an empty line between.

[eslint] There should be at least one empty line between import groups [import/order]